src/auth/logout.ts

```diff
@@ -0,0 +1,11 @@
+"use server";
+
+import { redirect } from "next/navigation";
+import { cookies } from "next/headers";
+
+// write logout endpoint
```

Comment:
I'm not sure you want to document your history on converstations with copilot ;D